psamm_import/importer.py

```diff
@@ -146,6 +162,7 @@ def detect_best_flux_limit(model):
     if len(flux_limit_count) == 0:
         return None
 
+    model.extracellular_compartment, _ = detect_extracellular(model)
```

Comment:
I think the function call to `detect_extracellular()` should be moved out of `detect_best_flux_limit()` since it is not relevant to detecting the flux limit. I would move it into `write_yaml_model()` after the place where `detect_best_flux_limit()` is called.